hc/api/transports.py

```diff
@@ -239,3 +241,15 @@ def notify(self, check):
         }
 
         return self.post(self.channel.value, payload)
+
+
+class Telegram(Transport):
+    def telegram_message(self, check):
```

Comment:
Use a descriptive method name. Kindly refer to `https://www.geeksforgeeks.org/java-naming-conventions/`